feg/gateway/tools/s6a_cli/main.go

```diff
@@ -272,7 +274,7 @@ func air(cmd *commands.Command, args []string) int {
 				errChann <- errCli
 				return
 			}
-			fmt.Printf("Sending AIR to %s:\n%s\n%+#v\n\n", peerAddr, json, *req)
+			fmt.Printf("Sending AIR to %s:\n%s\n%+#v\n\n", peerAddr, json, proto.Clone(req).(*protos.AuthenticationInformationRequest))
```

Comment:
It's crazy to me that this is necessary... does using `%+v` or `%v` resolve the need?

My assumption is fmt.Printf is copying the value of the passed ptr, which triggers the govet warning. Not sure why fmt.Printf is doing that copying though

Reply:
Actually, you are right. It is not necessary as we learned by now. Pointers can still be passed with `%+v` for example. It is fixed now.

Sorry, that this still appeared in the review. We didn't squash the fix up commits yet. Will do!